src/aws/lambda-encode/index.js

```diff
@@ -15,24 +19,29 @@ exports.handler = async (event, context, callback) => {
   const [resourceId, kind, recordId, extendedStamp] = parts;
   if (
     parts.length != 4 ||
-    !['timedtexttrack', 'video', 'thumbnail'].includes(kind)
+    !['document', 'thumbnail', 'timedtexttrack', 'video'].includes(kind)
   ) {
     let error;
     switch (kind) {
-      case 'video':
+      case 'document':
         error =
-          'Source videos should be uploaded in a folder of the form ' +
-          '"{playlist_id}/videos/{video_id}/{stamp}".';
+          'Source document should be uploaded to a folder of the form ' +
+          '"{document_id}/document/{document_id}/{stamp}".';
```

Comment:
the first parameter should be the playlist id?
```suggestion
          '"{playlist_id}/document/{document_id}/{stamp}".';
```

Reply:
No it's the document id. It's the same for the video, it's the video is first, you can check in the backend code ;-)

Reply:
ok! then the comment just above is wrong?
```python 
'"{playlist_id}/videos/{video_id}/{stamp}".';
```

Reply:
There should be an "s" to documents:
```suggestion
          '"{document_id}/documents/{document_id}/{stamp}".';
```

Reply:
I think yes. This is the code generating the source key:

```python
return "{pk!s}/video/{pk!s}/{stamp:s}".format(pk=self.pk, stamp=stamp)
```

As you can see the `id` is used and `video` is not in its plural form.

Reply:
yep. Let's just fix this comment then.